project/Dependencies.scala

```diff
@@ -23,6 +23,8 @@ object Dependencies extends CoreDependencies {
   def nlpstack(component: String) = ("org.allenai.nlpstack" % s"nlpstack-${component}_2.11" % "1.6")
     .exclude("commons-logging", "commons-logging")
 
+  val openCsv = "net.sf.opencsv" % "opencsv" % "2.1"
```

Comment:
I'm not sure we want this in the main common package . . . but it's probably fine.